### ISSUE TITLE
Controller alert methods should be extension methods rather than BootstrapBaseController

### ISSUE THREAD

**Issue by jarrettv** [erichexter/twitter.bootstrap.mvc#86]
I suggest Attention, Success, Information, and Error are made extension methods.

**Comment by erichexter** [comment]
I am cool including this as a pull request.

**Comment by serra** [comment]
:+1: 

I'm not sure what the `Update-Package twitter.bootstrap.mvc4` will do if `BootstrapBaseController` is unmodified. It might remove the file, possibly breaking client code. Please check this and handle it gracefully.

**Comment by erichexter** [comment]
So let me rephrase my support for this. Move the existing logic into
extension methods and change the existing controller base to call these
from the existing helper methods.
